data/human/hails.txt

```diff
@@ -2195,6 +2195,360 @@ phrase "friendly civilian"
+	word
+		"you've got to "
+		"you should "
+		"you should learn to "
+		"you're better off to "
+		"it's best to "
+		""
+	word
+		"avoid mining "
+		"avoid picking up "
+		"avoid harvesting "
+		"avoid "
+		"dump "
+		"jettison "
+	word
+		"silicon"
+		"iron"
+		"lead"
+		"aluminum"
+		"silicon ore"
+		"iron ore"
+		"lead ore"
+		"aluminum ore"
+		"minerals like silicon"
+		"minerals like iron"
+		"minerals like lead"
+		"minerals like aluminum"
+		"ore like silicon"
+		"ore like iron"
+		"ore like lead"
+		"ore like aluminum"
+	word
+		" and mine other asteroids"
+		" and find more valuable minerals"
+		" so you have space for more profitable minerals"
+		" because they don't sell for very much"
+
+phrase "miner advice: ship"
+	word
+		"you'll want to"
+		"it's a good idea to"
+		"it's advisable to"
+		"the best thing you can do is"
+	word
+		" "
+	word
+		"expand your fleet with"
+		"purchase"
+		"take out a mortgage for"
+		"buy"
+		"go to the shipyard and get"
+		"get"
+	word
+		" a "
+	word
+		"specialized mining ship like a Sunder"
+		"ship fit for mining, like a Sunder"
+		"Syndicate Sunder"
+		"Sunder and two mining drones"
+		"Sunder mining vessel"
+		"good mining ship, like a Sunder"
+		"defensible ship like a Headhunter"
+		"multi-purpose ship like a Headhunter"
+		"defensible ship like an Argosy"
+		"versatile ship like an Argosy"
+		"carrier like an Aerie. You can use the fighters to mine for you"
+		"squadron of Furies with mining lasers"
+		"ship with a large cargo hold"
+		"good freighter that can fit mining laser turrets"
+		"ship that can defend itself from pirates"
+		"powerful escort ship that can scare off pirates"
+		"ship that can fit loads of mining lasers"
+		"Behemoth loaded with mining laser turrets"
+
+phrase "miner advice: other"
+	word
+		"you should stick to patrolled systems so the pirates don't blow you up"
+		"you should stick to patrolled systems so you won't get extorted by pirates"
+		"you should avoid patrolled systems as they don't tend to have rare minerals"
+		"you should learn to recognize what asteroids are worthless or expensive"
+		"keep in mind that mining in systems frequented by pirates may be risky, but they often have good minerals"
+		"consider bribing pirates to leave you alone in systems with valuable asteroids"
+
+phrase "friendly civilian"
+	word
+		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
+		"Mining can be a good way to improve your piloting skills."
+		"Mining is a safe career option for new Sparrow pilots."
+		"Asteroid miners should be better appreciated by other captains. It takes a lot of effort to destroy asteroids and just as much effort to scoop them up into your hold."
+		"I'm sick and tired of other ships stealing the minerals I've mined before I can get to them."
+
+phrase "friendly civilian"
+	word
+		"I find that %"
+		"Often, %"
+		"While difficult, %"
+		"It looks difficult, but %"
```

Comment:
```suggestion
		"It looks difficult, but %"
		""
```
Isn't this why you have the replacements?

Reply:
Oops, I must have removed that by accident.